tests/fixtures/callback/dir_store/proxy/testhost.json

```diff
@@ -0,0 +1,278 @@
+{
+  "check_mode": false,
+  "host": "testhost",
+  "logs": [
+    {
+      "_ansible_no_log": false,
+      "changed": true,
+      "cmd": [
+        "echo",
+        "foo"
+      ],
+      "failed": false,
+      "invocation": {
+        "module_args": {
+          "_raw_params": "echo foo",
+          "_uses_shell": false,
+          "stdin_add_newline": true,
+          "strip_empty_ends": true,
+          "warn": false
+        }
+      },
+      "module": "command",
+      "name": "Changed task",
```

Comment:
not sure if that is also true for the old format (think not, as there everything is folded into the `messages` subentry), but I'm seeing it here, so commenting here.

what if the module *returns* a `name`, would that overwrite this entry? same for `module`. I think it's wrong to mix data coming from Ansible and the metadata *you* generate.

Reply:
Good point, we actually do the same for "name", "failed" and "module". Since this is the best time for incompatible change, let's perhaps introduce a subhash named "foreman" and store everything in there. Do you think "foreman" is safe enough? I think so, but I could also do "_foreman" instead.

Reply:
nothing *forbids* a module to return a `foreman` value, but the chances that this happens is way lower. and `_foreman` is probably even better.

Reply:
Actually this made me to completely change the format from scratch. It's now like this:

```json
 "results": [
    {
      "failed": false,
      "result": {
        "_ansible_no_log": false,
        "changed": true,
        "cmd": [
          "echo",
          "foo"
        ],
        // other keys from result dict
      },
      "task": {
        "action": "command",
        "any_errors_fatal": false,
        "async": 0,
        "async_val": 0,
        "uuid": "1c697a0b-ae03-9772-67e4-00000000000e"
        // other keys from task dict
      }
    }
  ]
```